src/Components/WebAssembly/JSInterop/src/WebAssemblyJSRuntime.cs

```diff
@@ -60,32 +61,52 @@ protected override void EndInvokeDotNet(DotNetInvocationInfo callInfo, in DotNet
             BeginInvokeJS(0, "DotNet.jsCallDispatcher.endInvokeDotNetFromJS", args, JSCallResultType.Default, 0);
         }
 
-        /// <inheritdoc />
```

Comment:
Ah, unless I'm missing something, I think we made a mistake in a previous recent change here. Sorry for not spotting this during code review before. Looks like the confusion was [here](https://github.com/dotnet/aspnetcore/pull/24640/files#r469861702).

It appears that https://github.com/dotnet/aspnetcore/pull/24640/files#diff-514554bd1120b4f06b0ba9fee70c3792 made a breaking change to this public type which we didn't plan for. The various public overloads of `InvokeUnmarshalled` were all removed and replaced by an explicit interface implementation.

I don't think there's any reason we should want to introduce a breaking change for those. When RC1 ships, we're likely to get a few confused customers asking what's going on as various 3rd-party libraries will break.

Can we please put back the same public `InvokeUnmarshalled` methods onto this type to restore compatibility with 3.2? It can still implement `IJSUnmarshalledRuntime` but those implementations should be regular public methods.

Reply:
Ah, good catch. I've made these public again.